Dockerfile

```diff
@@ -82,8 +82,8 @@ RUN mkdir -p ${APP_ROOT} && \
     curl -L https://github.com/ManageIQ/manageiq-ui-self_service/tarball/${REF} | tar vxz -C ${SSUI_ROOT} --strip 1
 
 ## Add ManageIQ source from local directory (dockerfile development) or from Github (official build)
-# ADD . ${APP_ROOT}
-RUN curl -L https://github.com/ManageIQ/manageiq/tarball/${REF} | tar vxz -C ${APP_ROOT} --strip 1
+ADD . ${APP_ROOT}
```

Comment:
@chriskacerguis

As I stated above the main reason against ADD is the "magical" part , meaning in some use cases is not straight forward, and this is reflected in the section you pasted, but this is not the case here.

In addition the  docker's best practices is obviously wrong at least about using ADD for tar files - exactly for the reason I mentioned 2 comments ago about (image size).
 
again bottom line I don't mind what is used here they are the same in this use case.
once the main issue is resolved (whether to accept the approach I'm suggesting) this can be changes easily.